src/operation-node/unique-constraint-node.ts

```diff
@@ -26,13 +26,13 @@ export const UniqueConstraintNode = freeze({
   },
 
   create(
-    columns: string[],
+    expressions: OperationNode[],
     constraintName?: string,
     nullsNotDistinct?: boolean,
   ): UniqueConstraintNode {
     return freeze({
       kind: 'UniqueConstraintNode',
-      columns: freeze(columns.map(ColumnNode.create)),
+      columns: freeze(expressions),
```

Comment:
Let's keep it matching the node property. Easier to digest.

```suggestion
    columns: OperationNode[],
constraintName?: string,
nullsNotDistinct?: boolean,
): UniqueConstraintNode {
return freeze({
kind: 'UniqueConstraintNode',
      columns: freeze(columns),
```